app/actions/websocket/threads.ts

```diff
@@ -11,7 +11,7 @@ export async function handleThreadUpdatedEvent(serverUrl: string, msg: WebSocket
 
         // Mark it as following
         thread.is_following = true;
-        processReceivedThreads(serverUrl, [thread], teamId);
+        await processReceivedThreads(serverUrl, [thread], teamId);
```

Comment:
why is the await needed here? I don't see this causing any effect

Reply:
True, it was unexpectedly added while I was going through all the places this function is used. Reverting it.